### PR TITLE
adding rhinofi-power-users

### PR DESCRIPTION
**Pull request by andyswx**
Adding Rhino.Fi Power User badge to playground.

### CODE REVIEW

File: group-generators/generators/rhinofi-power-users/index.ts [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  
+    generate: async (context: GenerationContext): Promise<GroupWithData[]> => {
+      return [
+        {
+          name: "rhinofi-power-users",
+          timestamp: context.timestamp,
+          data: {
+            "0xd69005F707ed755D530FbA0FBEa9eDe0749F9832": 1,
+            "0x0099F5326F05Fb8B3f541997526e9D9224a88147": 2,
+          },
```
Comment (leosayous21) [inline group-generators/generators/rhinofi-power-users/index.ts:20]: Here is missing the source account (this is a new notion that was added recently)
`accountSources: [AccountSource.ETHEREUM],`

**Comment by andyswx** [comment]
Thanks Leo, ill be sure to add it :-)

**Comment by leosayous21** [comment]
Thanks! 🙏

File: group-generators/generators/rhinofi-power-users/index.ts (hunk 1 above)
Comment (andyswx) [reply group-generators/generators/rhinofi-power-users/index.ts:20]: Thanks Leo, i've added AccountSource.ETHEREUM and pushed.

**Comment by leosayous21** [comment]
Hello @andyswx, all is green, do you want me to merge or your are waiting for more addresses in the group?

**Comment by andyswx** [comment]
Great, could you merge this PR and we'll do some testing on our side and
then we'll submit another PR with the expanded address list.
Thanks!

**Review by leosayous21 — APPROVED** [review]

**Comment by andyswx** [comment]
Thanks Leo!

**Comment by leosayous21** [comment]
It's deployed https://playground.sismo.io/ :-)

**Comment by andyswx** [comment]
Awesome, I’ll go check it out 😀

**Pull request merged**